ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/components/ReportParametersDetails.tsx

```diff
@@ -114,12 +116,23 @@ function ReportParametersDetails({
                             {getCVEsDiscoveredSinceText(formValues.reportParameters)}
                         </DescriptionListDescription>
                     </DescriptionListGroup>
-                    {isIncludeNvdCvssEnabled && formValues.reportParameters.includeNvdCvss && (
+                    {((isIncludeNvdCvssEnabled && formValues.reportParameters.includeNvdCvss) ||
+                        (isIncludeEpssProbabilityEnabled &&
+                            formValues.reportParameters.includeEpssProbability)) && (
```

Comment:
Nit that may be related to the way Github renders diffs: this was a little difficult to parse what was going on at first glance. What do you think about extracting the conditions to variables since each one is fairly verbose and used twice in this section?

Reply:
Yes, thank you. The change that you recommend makes it clearer how to add more properties.